soh/soh/Enhancements/randomizer/randomizer.cpp

```diff
@@ -4511,6 +4630,77 @@ void CreateIceTrapRandoMessages() {
                                               "Pour Noël, cette année, tu&n'auras que du %BCHARBON!&%rJoyeux Noël%w!" });
 }
 
+CustomMessageMinimal FireTempleGoronMessages[NUM_GORON_MESSAGES] = {
+    {
+        "Are you the one they call @?^You look really weird for %rDarunia's kid.%w&Are you adopted?",
+        "",
```

Comment:
Copy the english into these (with macro or manually) and leave a translation TODO comment